plugin/core/views.py

```diff
@@ -25,7 +25,12 @@ def get_line(window: Optional[sublime.Window], file_name: str, row: int) -> str:
 
 
 def point_to_offset(point: Point, view: sublime.View) -> int:
-    return view.text_point(point.row, point.col)
+    return view.text_point(
+        point.row,
+        # @see https://microsoft.github.io/language-server-protocol/specifications/specification-3-15/#position
+        # If the character value is greater than the line length it defaults back to the line length.
+        min(point.col, len(view.line(view.text_point(point.row, 0))))
+    )
```

Comment:
Can I poke you to write a test for this? tests/test_views.py seems like a good place to add some tests. `UnitTesting: Test Current File` is your friend :)

Reply:
I've added a test in https://github.com/sublimelsp/LSP/pull/904/commits/f45f90e5d0067ff33c3e7cc4f66d4a24f70d86a5. Thanks for the hint about `UnitTesting: Test Current File`.